packages/medusa/src/api/admin/customer-groups/middlewares.ts

```diff
@@ -64,4 +66,14 @@ export const adminCustomerGroupRoutesMiddlewares: MiddlewareRoute[] = [
       ),
     ],
   },
+  {
```

Comment:
I believe having `/admin/customers?customer_group=123` is better than having these relationship endpoints (and is the pattern we followed elsewhere, like price list products). 

We could always have this endpoint before releasing for backwards compatibility, but it should only reroute to the `/customers` endpoint. WDYT?

Reply:
Good push. I added this mainly to stay backward compatible, but I am fine removing them tbh. I don't imagine these endpoints are being used a lot, so maybe we should just introduce the breaking change. One less endpoint for us to main. Wdyt?

Reply:
@olivermrbl yeah I think we should go for it, and then when we think about migrations we look into what we want to keep as in v1 (and add a proxy for redirecting/remapping) and what we want to just go with a breaking change. But for now I think it's good to go from first principles and design the API as we want it.